app-localize-behavior.html

```diff
@@ -210,20 +246,30 @@
 
       // Cache the key/value pairs for the same language, so that we don't
       // do extra work if we're just reusing strings across an application.
-      var messageKey = key + resources[language][key];
-      var msg = proto.__localizationCache.messages[messageKey];
+      var value = resources[language][key];
```

Comment:
trnaslatedValue and translatedMessage sound good. The name "msg" was already being used, but I'll change it to "translatedMessage".